src/components/NurseryWithdrawals/WithdrawalDetails/sections/OutplantReassignmentTable.tsx

```diff
@@ -0,0 +1,69 @@
+import { Species } from 'src/types/Species';
+import { Delivery } from 'src/api/types/tracking';
+import { Table, TableColumnType } from '@terraware/web-components';
+import strings from 'src/strings';
+import { useEffect, useState } from 'react';
+
+const columns: TableColumnType[] = [
+  { key: 'species', name: strings.SPECIES, type: 'string' },
+  { key: 'from_plot', name: strings.FROM_PLOT, type: 'string' },
+  { key: 'to_plot', name: strings.TO_PLOT, type: 'string' },
+  { key: 'original_qty', name: strings.ORIGINAL_QTY, type: 'string' },
+  { key: 'final_qty', name: strings.FINAL_QTY, type: 'string' },
+  { key: 'notes', name: strings.NOTES, type: 'string' },
+];
+
+type OutplantReassignmentTableProps = {
+  species: Species[];
+  plotNames: Record<number, string>;
+  delivery?: Delivery;
+};
+
+export default function OutplantReassignmentTable({
+  delivery,
+  species,
+  plotNames,
+}: OutplantReassignmentTableProps): JSX.Element {
+  const [rowData, setRowData] = useState<{ [p: string]: unknown }[]>([]);
+
+  useEffect(() => {
+    // get list of distinct species
+    const speciesList =
+      delivery?.plantings?.reduce<number[]>(
+        (acc, pl) => (acc.includes(pl.speciesId) ? acc : [...acc, pl.speciesId]),
+        []
+      ) ?? [];
+    const rows: { [p: string]: unknown }[] = [];
+    for (const sp of speciesList) {
+      const speciesName = species?.find((x) => x?.id === sp)?.scientificName ?? '';
+      const plantings = delivery?.plantings?.filter((pl) => pl.speciesId === sp);
+      const deliveryPlanting = plantings?.find((pl) => pl.type === 'Delivery');
+      const reassignmentFromPlanting = plantings?.find((pl) => pl.type === 'Reassignment From');
+      const reassignmentToPlanting = plantings?.find((pl) => pl.type === 'Reassignment To');
+
+      // if reassignment plantings are found, create table rows
+      if (deliveryPlanting && reassignmentFromPlanting && reassignmentToPlanting) {
+        rows.push({
+          species: speciesName,
+          from_plot: '',
+          to_plot: deliveryPlanting.plotId ? plotNames[deliveryPlanting.plotId] : '',
+          original_qty: deliveryPlanting.numPlants.toString(),
+          final_qty: (deliveryPlanting.numPlants + reassignmentFromPlanting.numPlants).toString(),
+          notes: reassignmentToPlanting.notes ?? '',
```

Comment:
this is interesting, probably need to follow up on figma if we need to show the notes twice. for now we could leave it in for sure.

Reply:
from @rachel-tf-design:
> The notes for the first row is the withdrawal notes and the notes for the second row is the reassignment notes so they're different.

I will go ahead and make this change.